### PR TITLE
docs: Accept OEP-58 as it has been implemented

### PR DESCRIPTION
**Pull request by sarina**

### CODE REVIEW

File: oeps/architectural-decisions/oep-0058-arch-translations-management.rst [hunk 1]
```diff
@@ -18,15 +18,18 @@ OEP-58: Translations Management
...
    * - Created
      - 2022-08-08
    * - Review Period
      - 2022-11-07 - 2022-11-21
    * - References
-     - `Follow-up Work <https://openedx.atlassian.net/wiki/spaces/COMM/pages/3578494977/Translations+Management+Design+Implementation>`_
+     -
+       * `Concept doc about translations <https://docs.openedx.org/en/latest/translators/concepts/translation-process.html>`_
+       * `Enabling translations on a new repo <https://docs.openedx.org/en/latest/developers/how-tos/enable-translations-new-repo.html>`_
+       * `Follow-up Work <https://openedx.atlassian.net/wiki/spaces/COMM/pages/3578494977/Translations+Management+Design+Implementation>`_
```
Comment (sarina) [inline oeps/architectural-decisions/oep-0058-arch-translations-management.rst:32]: Should this link be left here? Has this wiki page been fully addressed or is there work left to do? I think there may be work on that page that has not been completed, but I don't see it as work that is blocking the acceptance of this OEP. Perhaps the wiki page could be narrowed to unimplemented details.

Comment (brian-smith-tcril) [reply oeps/architectural-decisions/oep-0058-arch-translations-management.rst:32]: I added comments to https://openedx.atlassian.net/wiki/spaces/COMM/pages/3578494977/Translations+Management+Design+Implementation

@OmarIthawi would you mind taking a look at that wiki page as well?

Comment (OmarIthawi) [reply oeps/architectural-decisions/oep-0058-arch-translations-management.rst:32]: Sure, I'll take a look. Sorry for keeping this stale.

Comment (OmarIthawi) [reply oeps/architectural-decisions/oep-0058-arch-translations-management.rst:32]: @sarina 

> Should this link be left here? Has this wiki page been fully addressed or is there work left to do? I think there may be work on that page that has not been completed, but I don't see it as work that is blocking the acceptance of this OEP. Perhaps the wiki page could be narrowed to unimplemented details.

The wiki are a bit outdated and the three of the links need more work.

I'm pushing a PR to make it less outdated by fixing things locally. We may still need to refactor the docs to become more digestable. 

Is it just me or the documentation work is hard?

**Review by OmarIthawi — APPROVED** [review]

**Pull request merged**